openssl-sys/build/main.rs

```diff
@@ -449,6 +449,8 @@ See rust-openssl documentation for more information:
             (3, 9, _) => ('3', '9', 'x'),
             (4, 0, 0) => ('4', '0', '0'),
             (4, 0, _) => ('4', '0', 'x'),
+            (4, 1, 0) => ('4', '1', '0'),
+            (4, 1, _) => ('4', '1', 'x'),
             _ => version_error(),
```

Comment:
`version_error` needs to be udpated to reflect the new version. Sorry I missed this earlier!

Reply:
Of course. Sorry about that. Fixed.